vdirsyncer/storage/dav.py

```diff
@@ -1,7 +1,15 @@
 # -*- coding: utf-8 -*-
 
+from __future__ import print_function
```

Comment:
No, python2 is not unsupported, but IIRC, this line would only be needed for python2 versions we don't support (ie: python26?).